build/citations.py

```diff
@@ -61,13 +61,10 @@ def get_text(directory):
     Return a dictionary of section texts in the specified directory.
     """
     section_dir = pathlib.Path(directory)
-    paths = sorted(section_dir.iterdir())
+    paths = sorted(section_dir.glob('[0-9]*.md'))
```

Comment:
The (existing) requirement that manuscript sections begin with numbers makes sense to me.  This is something else to document in case someone expects all `.md` files to be included.